pkg/git/libgit2/managed/ssh.go

```diff
@@ -59,6 +59,7 @@ import (
 	"golang.org/x/crypto/ssh"
 	"golang.org/x/net/proxy"
 
+	"github.com/distribution/distribution/v3/uuid"
```

Comment:
Maybe we should use `github.com/google/uuid` instead? The whole package is specifically made for uuid and we already it in our dependencies.